### PR TITLE
Multiple Issue templates

### PR DESCRIPTION
**Pull request by ykzts**
ref https://blog.github.com/2018-05-02-issue-template-improvements/

### Screenshot

screenshot

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-[Issue text goes here].
+---
+name: Bug Report
+about: Create a report to help us improve
+
+---
+
+<!-- Issue text goes here. -->
```
Comment (nightpool) [inline .github/ISSUE_TEMPLATE/bug_report.md]: i'm worried about people putting the issue text inside of the comment on accident, leading to very confusing "empty" issues. Can we go back to [Issue text goes here]?

**Review by nightpool — CHANGES_REQUESTED** [review]
LGTM w/o comments

**Comment by nightpool** [comment]
Maybe we want another template for "I'm having an issue running Mastodon" or similar, for admin support stuff?

**Comment by ykzts** [comment]
@nightpool 👍

**Review by nightpool — APPROVED** [review]

**Comment by nightpool** [comment]
I think you misinterpreted what I said, but i'm fine w/o it, I was just thinking a third template for admins might be nice, but i don't know what we'd put there.......

**Review by Gargron — APPROVED** [review]

**Pull request merged**